### ISSUE TITLE
ProgressBar - strange behavior

### ISSUE THREAD

**Issue by MasterzYuri** [DmitryMalkovich/circular-with-floating-action-button#7]
This is what happens when I tap the FAB button: [Video]( https://vid.me/RGxx)

    public class ImagesAdapter extends Fragment {

       //...\\

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup parent, Bundle savedInstanceState) {
        View rootView = inflater.inflate(R.layout.image_adapter, parent, false);

        coordinatorLayout = (CoordinatorLayout) rootView.findViewById(R.id.coordinator_layout);
        progressFloatingActionButton = (ProgressFloatingActionButton) rootView.findViewById(R.id.fab);
        progressBar = (ProgressBar) rootView.findViewById(R.id.progressBar);
        if (progressBar != null) {        
            progressBar.setVisibility(View.INVISIBLE);
        }

        attachListeners();
        return rootView;
    }

    private void attachListeners() {
        progressFloatingActionButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                progressBar.setVisibility(View.VISIBLE);   
        
                //....download the image...\\

                getActivity().registerReceiver(onComplete, new IntentFilter(DownloadManager.ACTION_DOWNLOAD_COMPLETE));
            }
        });
    }

    BroadcastReceiver onComplete=new BroadcastReceiver() {
        public void onReceive(Context ctxt, Intent intent) {
            progressBar.setVisibility(View.INVISIBLE);
            Snackbar snackbar = Snackbar
                    .make(coordinatorLayout, "Image saved!", Snackbar.LENGTH_SHORT);
            View snackBarView = snackbar.getView();          
            TextView textView = (TextView) snackBarView .findViewById(android.support.design.R.id.snackbar_text);
            textView.setTextSize(16);
            snackbar.show();
          }
       };
    }

```
<?xml version="1.0" encoding="utf-8"?>
<android.support.design.widget.CoordinatorLayout
        xmlns:android="http://schemas.android.com/apk/res/android"
         xmlns:app="http://schemas.android.com/apk/res-auto"
        android:layout_width="match_parent"
        android:layout_height="match_parent"
        android:id="@+id/coordinator_layout">

        <ImageView
            android:id="@+id/image_pager"
            android:layout_width="match_parent"
            android:layout_height="match_parent"
            android:scaleType="fitCenter"
            android:adjustViewBounds="true"/>

        <com.dmitrymalkovich.android.ProgressFloatingActionButton
            android:id="@+id/fab"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            android:layout_gravity="bottom|end"
            android:layout_margin="@dimen/fab_margin">

            <android.support.design.widget.FloatingActionButton
                android:layout_width="wrap_content"
                android:layout_height="wrap_content"
                android:src="@drawable/ic"
                app:backgroundTint="@color/primary" />

            <ProgressBar
                android:id="@+id/progressBar"
                style="@style/Widget.AppCompat.ProgressBar"
                android:layout_width="wrap_content"
                android:layout_height="wrap_content"/>

        </com.dmitrymalkovich.android.ProgressFloatingActionButton>

    </android.support.design.widget.CoordinatorLayout>
```

**Comment by DmitryMalkovich** [comment]
Hello @MasterzYuri, thank you for creating the issue. I will try to fix it asap!

**Comment by DmitryMalkovich** [comment]
Fixed in 1.5. Could you try it with your app?
